lib/registry.ts

```diff
@@ -291,7 +292,18 @@ export class Registry {
   }
 
   public resolveBeforeHooks(tags: string[]) {
-    return this.resolveCaseHooks("Before", tags);
+    return this.resolveCaseHooks("Before", tags).sort(
+      (a: ICaseHook, b: ICaseHook) => {
+        // If order is not specified, the hook will be executed at the end
+        if (a.order === undefined) {
+          a.order = Number.MAX_SAFE_INTEGER;
```

Comment:
Default order should be made to 10000 in order to mimic behavior seen in other implementations, ref. https://github.com/cucumber/cucumber-jvm/commit/1a5e98db62ed11c9ad151dd81219b8782e27b3f8.

Reply:
Done